CHANGELOG.md

```diff
@@ -19,6 +19,8 @@
   * BTDropInResult
     * Rename `BTUIKPaymentOptionType` enum to `BTDropInPaymentMethodType`
     * Rename `paymentOptionType` property to `paymentMethodType`
+  * BTDropInPaymentMethodType
+    * Remove `BTDropInPaymentMethodTypeCoinbase`
```

Comment:
I'm not sure what's the least confusing, but we could combine this with one of the lines above. Something like:

```
* Rename `BTUIKPaymentOptionType` enum to `BTDropInPaymentMethodType` and remove the `.coinbase` case 
```
I'm just thinking that _technically_ `BTDropInPaymentMethodTypeCoinbase` never existed in a released version of the SDK, so it's maybe a little confusing to say that we're removing it.

Reply:
That's a good callout 🤔